features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/async/RecordedDataQueueHandler.kt

```diff
@@ -133,10 +140,19 @@ internal class RecordedDataQueueHandler {
                 triggerProcessingLoop(currentTime)
             }
         } catch (e: RejectedExecutionException) {
-            // TODO: REPLAY-1364 Add logs here once the sdkLogger is added
+            internalLogger.log(
```

Comment:
minor: since we're using essentially the same parameters for each of these, perhaps we could extract to a method and pass the exception